.travis.yml

```
@@ -0,0 +1,2 @@
+language: 'node_js'
+node_js: 'current'
```

You're also going to need something like this for Travis to recognize your instance of Mongo: 
```suggestion
node_js: 'current'
services:
  - mongodb
env:
  - MONGODB_URI=mongodb://localhost:27017/tweets AUTH_SECRET=MONKEYBANANASPLITS
```